sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/TungstenAggregate.scala

```diff
@@ -69,6 +72,9 @@ case class TungstenAggregate(
   protected override def doExecute(): RDD[InternalRow] = attachTree(this, "execute") {
     val numInputRows = longMetric("numInputRows")
     val numOutputRows = longMetric("numOutputRows")
+    val totalPeakMemory = longMetric("totalPeakMemory")
```

Comment:
I see, given limited space, I question the utility of this metric (these tasks might not have even been running at the same time).  The percentiles seem way more useful (i.e. is skew causing some partitions to spill or are they all spilling).  We can leave it in if @pwendell and @rxin feel strongly though.